imports/ui/pages/admin-panel/admin-panel.html

```diff
@@ -0,0 +1,79 @@
+<template name='adminPanel'>
+	<div class="container-sm">
+		{{#if isAdmin}}
+			<h1>{{mf 'adminPanel.heading' 'Admin panel'}}</h1>
+			{{#if activeTask}}
+				<button type="button" class="btn btn-link" id="backToDashboard">
+					<i class="fa fa-chevron-left"></i>
+					{{mf 'adminPanel.backToDashboard' 'Go back to dashboard'}}
+				</button>
+				<div class="admin-active-task">
+					{{> Template.dynamic template=activeTask}}
+				</div>
+			{{else}}
+				{{> adminDashboard}}
+			{{/if}}
+		{{else}}
+			please login
+		{{/if}}
+	</div>
+</template>
+
+<template name="adminDashboard">
+	<div class="admin-dashboard">
+		{{#each tasks}}
+			<div class="admin-task-wrap">
+				<div class="admin-task js-admin-task">
+					<i class="fa fa-2x {{icon}}" aria-hidden="true"></i>
+					<h3>{{name}}</h3>
+				</div>
+			</div>
+		{{/each}}
+	</div>
+</template>
+
+<template name="featureGroup">
+	{{#if featuredGroup}}
+		<div class="admin-featured-group">
+			<div class="admin-featured-group-body">
+				{{#mf KEY='featureGroup.groupIsFeatured' GROUP=featuredGroup.name REGION=regionName}}
+					The group "{GROUP}" is featured for the region "{REGION}".
+				{{/mf}}
+			</div>
+			<button type="button" id="unsetFeaturedGroup" class="btn btn-delete" {{disabledIfBusy}}>
+				{{#if busy 'deleting'}}
+					<i class="fa fa-circle-o-notch fa-spin fa-fw" aria-hidden="true"></i>
+					{{mf '_button.deleting'}}
+				{{else}}
+					<i class="fa fa-trash fa-fw" aria-hidden="true"></i>
+					{{#mf KEY='featureGroup.deleteButton' NAME=regionName}}
+						Remove the featured group for "{NAME}"
+					{{/mf}}
+				{{/if}}
+			</button>
+		</div>
+	{{/if}}
+	<h3>{{mf 'featureGroup.featureGroup' 'Feature group'}}</h3>
+	<form id="featureGroup">
+		<div class="form-group">
+			<select id="groupToBeFeatured" class="form-control">
+				{{#each group in groups}}
+					<option value="{{group._id}}">{{group.name}}</option>
```

Comment:
This will become very long. For now we can just wait until somebody complains :-)